src/Crowbar.ycp

```diff
@@ -51,27 +51,29 @@ string provisioner_file  = "/etc/crowbar/provisioner.json";
 string installed_file    = "/opt/dell/crowbar_framework/.crowbar-installed-ok";
 
 /**
- * Names of default repositories available for current product
+ * The keys are the names of default repositories available for current product
  * For Cloud5, we support mixed SLE11/SLE12 environments
+ *
+ * The values are target node platform for those repositories
  */
-list<string> default_repos       = [
-  "SLE-Cloud",
-  "SLE-Cloud-PTF",
-  "SUSE-Cloud-5-Pool",
-  "SUSE-Cloud-5-Updates",
-  "SLES11-SP3-Pool",
-  "SLES11-SP3-Updates",
-  "SLE11-HAE-SP3-Pool",
-  "SLE11-HAE-SP3-Updates",
-  "SLES12-Pool",
-  "SLES12-Updates",
-  "SLE12-Cloud-Compute",
-  "SLE12-Cloud-Compute-PTF",
-  "SLE12-Cloud-5-Compute-Pool",
-  "SLE12-Cloud-5-Compute-Updates"
+map<string,string> default_repos       = $[
+  "SUSE-Cloud-5-Pool"                   : "common",
+  "SUSE-Cloud-5-Updates"                : "common",
+  "SLE-Cloud"                           : "suse-11.3",
+  "SLE-Cloud-PTF"                       : "suse-11.3",
+  "SLES11-SP3-Pool"                     : "suse-11.3",
+  "SLES11-SP3-Updates"                  : "suse-11.3",
+  "SLE11-HAE-SP3-Pool"                  : "suse-11.3",
+  "SLE11-HAE-SP3-Updates"               : "suse-11.3",
+  "SLES12-Pool"                         : "suse-12.0",
+  "SLES12-Updates"                      : "suse-12.0",
+  "SLE12-Cloud-Compute"                 : "suse-12.0",
+  "SLE12-Cloud-Compute-PTF"             : "suse-12.0",
+  "SLE12-Cloud-5-Compute-Pool"          : "suse-12.0",
+  "SLE12-Cloud-5-Compute-Updates"       : "suse-12.0",
+  "SUSE Enterprise Storage"             : "suse-12.0"
```

Comment:
And this should be SUSE-Enterprise-Storage-1.0-Pool and SUSE-Enterprise-Storage-1.0-Updates